docs/source/index.rst

```diff
@@ -10,8 +10,9 @@ In each of the sections below, you will find overviews, guides, and step-by-step
 the features of MLflow and how they can be leveraged to solve real-world MLOps problems. 
 
 
-Getting Started with MLflow
----------------------------
+`Getting Started with MLflow <getting-started/index.html>`_
+-----------------------------------------------------------
```

Comment:
The alternative is to create reference labels, which are a major source of pain for avoiding collisions when building the docs. Based on what I've been finding and reading in forums / commentary, a lot of larger docs contributors avoid those link reference tags due to this frustration and instead do relative path declaration, as shown above. It helps to simplify redirects instead of having to hunt for declared ref labels.